File: src/main/java/ContainerInit.java
import com.github.mikn.better_seeds.BetterSeeds;
import net.minecraft.world.inventory.MenuType;
import net.minecraftforge.registries.DeferredRegister;
import net.minecraftforge.registries.ForgeRegistries;
import net.minecraftforge.registries.RegistryObject;

public final class ContainerInit {
    public static final DeferredRegister<MenuType<?>> CONTAINERS = DeferredRegister.create(ForgeRegistries.CONTAINERS,
            BetterSeeds.MOD_ID);

    public static final RegistryObject<MenuType<MyChestContainer>> EXAMPLE_CHEST = CONTAINERS
            .register("example_chest", () -> new MenuType<>(MyChestContainer::new));

    private ContainerInit() {
    }
}

File: src/main/kotlin/com/github/mikn/better_seeds/init/ContainerInit.kt
package com.github.mikn.better_seeds.init

import com.github.mikn.better_seeds.BetterSeeds
import com.github.mikn.better_seeds.container.SeedModifierBlockContainer
import net.minecraftforge.common.extensions.IForgeMenuType
import net.minecraftforge.registries.DeferredRegister
import net.minecraftforge.registries.ForgeRegistries

class ContainerInit {
    companion object {
        @JvmStatic
        val CONTAINERS = DeferredRegister.create(ForgeRegistries.CONTAINERS, BetterSeeds.MOD_ID)
        @JvmStatic
        val SEED_MODIFIER_CONTAINER = CONTAINERS.register("seed_modifier") {
            IForgeMenuType.create { windowId, inv, _ -> SeedModifierBlockContainer(windowId, inv) }
        }
    }
}